roles/zabbix_agent/tasks/api.yml

```diff
@@ -7,6 +7,7 @@
     - zabbix_api_create_hostgroup | bool
   register: zabbix_api_hostgroup_created
   until: zabbix_api_hostgroup_created is succeeded
+  retries: 10
```

Comment:
I have added 10 retries to the API calls, to workaround the connection problems when running Ansible on macOS.
This change does not hurt anyone. Indeed this is a better solution than a `throttle`.